packages/ia-activity-indicator/index.js

```diff
@@ -1,5 +1,5 @@
-import { IAActivityIndicator } from './src/ia-activity-indicator.js';
+import { IAActivityIndicator, IAActivityIndicatorMode } from './src/ia-activity-indicator.js';
 
 window.customElements.define('ia-activity-indicator', IAActivityIndicator);
 
-export { IAActivityIndicator, IAActivityIndicatorMode } from './src/ia-activity-indicator.js';
+export { IAActivityIndicator, IAActivityIndicatorMode };
```

Comment:
Q: I'm mostly just curious why we needed to register the element here in `index.js` at all. Shouldn't it get registered automatically by virtue of extending `LitElement`?

Reply:
In our TS repos, you can register with the `@customElement('ia-activity-indicator')` decorator, but this is a JS package so you have to manually register with `window.customElements.define('ia-activity-indicator', IAActivityIndicator);`

This is a pretty old Lit package (maybe one of our first?) and they used to recommend registering the component independently of the actual component definition so we had this separate registration file. Now it's usually just bundled together in the same file.

Reply:
I moved the registration into the element itself so we get the registration regardless of how it gets imported.